package.json

```
@@ -48,7 +48,7 @@
     "test:search-indexing": "jest ./search-indexing",
     "test:server": "cd ./api-server && npm test && cd ../",
     "test:tools": "jest ./tools",
-    "test:utils": "jest ./utils/*.js"
+    "test:utils": "bash -c 'jest ./utils/*.js'"
```

```suggestion
    "test:utils": "jest --rootDir ./utils"
```

bash could be absent on windows

---

@ValeraS Works great!  Thanks for the suggestion.